RevenueCatUI/Views/RevenueCatDeeplinkHandler.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  RevenueCatDeeplinkHandler.swift
+//
+//  Created by Andrés Boedo on 9/17/24.
+
+import Foundation
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable, message: "RevenueCatUI does not support macOS yet")
+@available(tvOS, unavailable, message: "RevenueCatUI does not support tvOS yet")
+struct RevenueCatDeeplinkHandlerView<Content: View>: View {
```

Comment:
to get this through the finish line, we really need to: 
- settle on the exact format. I used `/paywall` in the examples but I'm not even enforcing it, maybe `revenuecatui` is more appropriate? 
- clean up error handling and logging
- add tests